AudioKit/Common/Internals/Audio File/AKAudioFile+Async Processing.swift

```diff
@@ -0,0 +1,365 @@
+//
+//  AKAudioFile+processing Async.swift
```

Comment:
This title doesn't match the filename.  Also, there should be no spaces in filenames.